charts/consul/templates/connect-inject-deployment.yaml

```diff
@@ -252,9 +274,11 @@ spec:
             successThreshold: 1
             timeoutSeconds: 5
           volumeMounts:
+          {{- if not (and .Values.global.secretsBackend.vault.enabled .Values.global.secretsBackend.vault.connectInject.tlsCert.secretName) }}
```

Comment:
don't need to mount these when using vault because they are used in webhook-cert-manager mode.  vault downloads the certs to `/vault/secrets`.